sui/src/config.rs

```diff
@@ -176,17 +176,19 @@ impl NetworkConfig {
                 .map(|x| {
                     let name = x.key_pair.make_narwhal_keypair().name;
                     let primary = PrimaryAddresses {
-                        primary_to_primary: format!("{}:{}", x.host, x.port + 1).parse().unwrap(),
-                        worker_to_primary: format!("{}:{}", x.host, x.port + 2).parse().unwrap(),
+                        primary_to_primary: format!("{}:{}", x.host, x.port + 100).parse().unwrap(),
+                        worker_to_primary: format!("{}:{}", x.host, x.port + 200).parse().unwrap(),
                     };
                     let workers = [(
                         /* worker_id */ 0,
                         WorkerAddresses {
-                            primary_to_worker: format!("{}:{}", x.host, x.port + 3)
+                            primary_to_worker: format!("{}:{}", x.host, x.port + 300)
                                 .parse()
                                 .unwrap(),
                             transactions: x.consensus_address,
-                            worker_to_worker: format!("{}:{}", x.host, x.port + 4).parse().unwrap(),
+                            worker_to_worker: format!("{}:{}", x.host, x.port + 400)
+                                .parse()
+                                .unwrap(),
```

Comment:
Given that there is a conflict, couldn't we just change the `x.port` parameter? E.x if it runs on `12000` maybe make it run on `10200` , so we avoid touching those here. Can't evaluate at the moment whether that would fix the issue, but I am thinking that we might get same effect with minimal changes.

Reply:
I am not sure how the docker files are written (perhaps they open a range of ports so changing the base may be a problem). I just tried to fix some errors that people have been seeing in the simplest way. All of this need to be properly fixed. I will bring the discussion during a next meeting.

Reply:
Cool, so this (ugly) fix should work for everything we need for DevNet if I get it right

Reply:
@asonnino still not clear on what the original conflict is.  I don't think you would get conflicts from Docker itself, can you clarify how you ran into the original conflicts?   In which environment, and what do you mean by running a cluster locally?   If it is the standard `Sui validator` command you just need +10 separation not +100....

Reply:
Yes a separation of +10 should work. Sadly I don't remember exactly what was the problem (I have been extinguishing fires for the last week). I think there was a problem with tracing + one primary trying to boot on the same port as a Sui authority